src/utils/handleError.ts

```diff
@@ -0,0 +1,9 @@
+import { Errors } from './Errors';
+
+export const handleError = (
+  error: Errors,
+  setErrorMessage: (error: Errors) => void,
+) => {
+  setErrorMessage(error);
+  setTimeout(() => setErrorMessage(Errors.Default), 3000);
```

Comment:
It would be best if you had a clear timer because it сreates additional accounts

```
const timeoutId = setTimeout(() => {
    setErrorMessage(Errors.Default);
  }, 3000);

  return () => clearTimeout(timeoutId);
```

Reply:
Good point. Do it